msal-python-conceptual/advanced/macos-broker.md

```diff
@@ -12,12 +12,12 @@ ms.author: ddelimarsky
 ms.reviewer: shermanouko, rayluo
 ---
 
-# Using MSAL Python with Authentication Brokers on macOS
+# Using MSAL Python with an Authentication Broker on macOS
 
 >[!NOTE]
 >macOS authentication broker support is introduced with `msal` version 1.31.0.
 
-Using authentication brokers on macOS enables you to simplify how your users authenticate with Microsoft Entra ID from your application, as well as take advantage of advanced functionality such as **token binding**, protecting any issued tokens from exfiltration and misuse.
+Using an authentication brokers on macOS enables you to simplify how your users authenticate with Microsoft Entra ID from your application, as well as take advantage of future functionality that protects Microsoft Entra ID tokens from exfiltration and misuse.
```

Comment:
We should either say Microsoft Entra ID refresh tokens, or not say anything at all :) Don't want someone to assume this is a promise that access tokens will also be protected